Dockerfile

```diff
@@ -28,8 +28,10 @@ RUN apk --update --no-cache add \
     git \
     graphviz \
     imagemagick \
+    libldap \
     monitoring-plugins \
     mtr \
+    musl \
```

Comment:
Not required since `php7-ldap` depends of

Reply:
Removed